src/plugins/build.ts

```diff
@@ -22,6 +22,30 @@ export function BuildPlugin(ctx: PWAPluginContext) {
         return injectServiceWorker(html, options, false)
       },
     },
+    async writeBundle() {
+      // add support for new SvelteKit Vite Plugin
+      const {
+        options: {
+          disable,
+          svelteKitVitePluginOptions: {
+            prerenderTimeout = 0,
+            disabled = false,
+          },
+        },
+        viteConfig,
+      } = ctx
+
+      const sveltekitPresent = disabled || disable || !viteConfig.build.ssr
```

Comment:
why have two disable options?

Reply:
The disabled option should be for disable the writeBundle.